system/lib/libcxxabi/include/cxxabi.h

```diff
@@ -86,7 +85,7 @@ extern _LIBCXXABI_FUNC_VIS _LIBCXXABI_NORETURN void __cxa_pure_virtual(void);
 extern _LIBCXXABI_FUNC_VIS _LIBCXXABI_NORETURN void __cxa_deleted_virtual(void);
 
 // 3.3.2 One-time Construction API
-#if defined(__arm__) || defined (__EMSCRIPTEN__)
+#if defined(_LIBCXXABI_GUARD_ABI_ARM)
```

Comment:
looks like we lost an `__EMSCRIPTEN__` guard here?

Reply:
See the comment in tools/system_libs.py.  I defined _LIBCXXABI_GUARD_ABI_ARM on the command line.   We can remove that once we roll to 10.0.0 since its defined upsteam now for wasm:
https://github.com/llvm-mirror/libcxxabi/blob/master/include/__cxxabi_config.h#L75

Reply:
Seems like a reasonable workaround, I guess, but confusing since we don't use the ARM ABI (or do we?). But I guess there's no simpler way?

Reply:
I'm not changing this behaviour as part of this change.  We were doing this both before and after.   IIRC we did do it (and have always done it) this way to save space, and I landed this change upstream llvm to be consistent with out ABI.   

See: https://reviews.llvm.org/D64961